GETTING_STARTED.md

```diff
@@ -987,6 +987,8 @@ namespace :factory_girl do
   task lint: :environment do
     if Rails.env.test?
       begin
+        DatabaseCleaner.strategy = :transaction
```

Comment:
Do we need to specify the strategy? It looks like DatabaseCleaner [tries to infer this](https://github.com/DatabaseCleaner/database_cleaner/blob/14626296ef31b34f7704593667db1d41d81825ab/lib/database_cleaner/base.rb#L179)